### PR TITLE
Blogpost: Julia Package Development Seminar

### PR DESCRIPTION
**Pull request by samumantha**
Do we want to add short seminar reports to the webpage as blogs?
The suggestion attached here is not ready to be posted, please lets make it nice together :)
Feel free to add/comment/edit/rewrite.

This would not replace anything more content wise if @lucaferranti wants to write something

### CODE REVIEW

**Comment by samumantha** [comment]
Could it be the `/)`  in the end of the link introducing the failing build?

**Comment by samumantha** [comment]
or a space after the `+++`?

**Comment by lucaferranti** [comment]
after removing the space after `+++` the site builds locally on my machine

File: content/blog/2021-08-18-seminar_report_julia_package.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
++++
```
Comment (lucaferranti) [inline content/blog/2021-08-18-seminar_report_julia_package.md:1]:
```suggestion
+++
```

**Comment by samumantha** [comment]
If there is no comments or improvement suggestions, should this be published?

File: content/blog/2021-08-18-seminar_report_julia_package.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+[extra]
+
+author = "Nordic-RSE" 
++++
+
+# Seminar report: Package development in Julia
+
+On August 18th 2021 we started our (to be) monthly Research Software Seminar Series with Luca Ferranti from University of Vaasa giving insights in Package development with Julia.
+9 interested learners + host and speaker joined the zoom call for about 3 hours. Luca had set up the session in a way that everyone who wants can type along with him. All material can be found in [this github repository](https://github.com/lucaferranti/juliaPackageDevelopment/) and the Agenda and Q&A [here](https://hackmd.io/@nordic-rse/package-development-julia).
+Topics captured were the Julia REPL and how to use and create environments. Luca showed us his general workflow for creating packages from scratch, which gave a good startingpoint for people to develop their own Julia packages.
```
Comment (lucaferranti) [inline content/blog/2021-08-18-seminar_report_julia_package.md:13]:
```suggestion
Topics captured were the Julia REPL and how to use and create environments. Luca showed us his general workflow for creating packages from scratch, which gave a good starting point for people to develop their own Julia packages.
```

**Comment by lucaferranti** [comment]
I left a small typo fix, otherwise LGTM

**Review by lucaferranti — APPROVED** [review]

**Comment by samumantha** [comment]
Thanks, if there is no more comments, please someone merge :)

**Pull request merged**